syft/execution/plan.py

```diff
@@ -159,7 +162,7 @@ def parameters(self):
         else:
             return []
 
-    def build(self, *args):
+    def build(self, *args, trace_autograd=False):
```

Comment:
Hum, I'm not sure having a trace_autograd attribute and a trace_autograd arg in build is the best solution. I think that you did that because of __call__ but maybe both could use the attribute? Or __call__ could also take it as an argument even if it seems odd

Reply:
`trace_autograd` is Plan build option, so I added it as kward in `build` method.
And also it can be specified with decorator. So you can specify it when building the Plan both ways.